test/features/support/ui_oauth_details.js

```diff
@@ -0,0 +1,199 @@
+  async () => await errorResultFunction("invalid_scope")
+);
+
+// Scenario: The user is not able to validate the provided request parameters and resolve the required authentication factors because of an invalid clientId parameter
+Given(
+  "The user wants to validate the provided request parameters and resolve the required authentication factors with an invalid clientId parameter",
+  () => (invalidClientId = "")
+);
+
+When("The user triggers an action with an invalid clientId parameter", () =>
+  requestFunction(
+    validScope,
+    validResponseType,
+    invalidClientId,
+    validRedirectUri
+  )
+);
+
+Then(
+  "The result of an operation returns an error, because of an invalid clientId provided",
+  async () => await errorResultFunction("invalid_client_id")
+);
+
+// Scenario: The user is not able to validate the provided request parameters and resolve the required authentication factors because of an invalid responseType provided
+Given(
+  "The user wants to load the JS application and validates the provided query parameters using the OAuth-details endpoint with an invalid responseType parameter",
+  () => (invalidResponseType = "Invalid response type")
+);
+
+When("The user triggers an action with an invalid responseType parameter", () =>
+  requestFunction(
+    validScope,
+    invalidResponseType,
+    validClientId,
+    validRedirectUri
+  )
+);
+
+Then(
+  "The result of an operation returns an error because of an invalid responseType provided",
+  async () => await errorResultFunction("invalid_response_type")
+);
+
+// Scenario: The user is not able to validate the provided request parameters and resolve the required authentication factors because of an invalid redirectUri provided
+Given(
+  "The user wants to validate the provided request parameters and resolve the required authentication factors with an invalid redirectUri parameter",
+  () => (invalidRedirectUri = "")
+);
+
+When("The user triggers an action with an invalid redirectUri parameter", () =>
+  requestFunction(
+    validScope,
+    validResponseType,
+    validClientId,
+    invalidRedirectUri
+  )
+);
+
+Then(
+  "The result of an operation returns an error because of an invalid redirectUri provided",
+  async () => await errorResultFunction("invalid_redirect_uri")
+);
+
+// Scenario: The user is not able to validate the provided request parameters and resolve the required authentication factors because none parameters provided
+Given(
+  "The user wants to validate the provided request parameters and resolve the required authentication factors without parameters",
+  () => "None of the required parameters was provided."
+);
+
+When("The user triggers an action without the required parameters", () =>
+  specUIOAuthDetails.post(baseUrl).withBody({})
+);
+
+Then(
+  "The result of an operation returns an error because none required parameters provided",
+  async () => await errorResultFunction("invalid_body")
```

Comment:
![image](https://user-images.githubusercontent.com/119664031/214904245-e32cc90e-f67a-4eb0-bea4-53fa8f72c12c.png)

There is no ```invalid_body``` in errors enum